iChef-WebApi/src/main/java/com/kitchen/iChef/Service/UserService.java

```diff
@@ -0,0 +1,38 @@
+package com.kitchen.iChef.Service;
+
+import com.kitchen.iChef.Domain.AppUser;
+import com.kitchen.iChef.Repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
```

Comment:
I know that it might be weird but we should have an interface for the service as well so that it can be faked in tests if needed.

Reply:
What I mean is literally create an IUserService interface, no need for a generic one.

Reply:
talked with Backend team, decided to not use Interfaces for services